extra/mtls/certs/client/cert.conf

```diff
@@ -0,0 +1,12 @@
+[req]
```

Comment:
File committed by mistake? Is it also generated in the script

Reply:
Actually, I wanted it to be in thee commit to have the right configuration used to generate the certs together with the certs. Is it a bad idea?

Reply:
I don't know. This file is overridden three times by the script, so why save only one of them? And why not to save also the server conf file?

These kind of questions makes me thing that it is best not to have it committed.

Reply:
ok, let's remove them!